test_rclcpp/CMakeLists.txt

```diff
@@ -13,6 +13,9 @@ endif()
 find_package(ament_cmake REQUIRED)
 
 if(BUILD_TESTING)
+  find_package(rclcpp REQUIRED)
```

Comment:
I am a bit confused how this package even passes CMake without this. The `ament_target_dependencies(... rclcpp)` calls below all require that `rclcpp` was found before :thinking: